apiary-metastore-events/apiary-metastore-consumer/privileges-grantor/privileges-grantor-core/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.expediagroup.apiary</groupId>
+        <artifactId>apiary-privileges-grantor-parent</artifactId>
+        <version>4.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apiary-privileges-grantor-core</artifactId>
+
+    <properties>
+        <!-- Plugins -->
+        <mavenSurefire.version>2.22.0</mavenSurefire.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.expediagroup.apiary</groupId>
+            <artifactId>apiary-hive-consumer-common</artifactId>
```

Comment:
This is pulling in the old dependency which you have renamed, you probably haven't noticed it locally as I'm guessing you still have that folder locally or the artifact in your local Maven cache. If you do a fresh checkout of this branch it fails to compile. I'm fixing this now.

Reply:
great find, thanks @massdosage